configs/permissions/inventory.json

```diff
@@ -0,0 +1,4 @@
+{
+    "*": ["*"],
+    "host_data": ["read", "write"]
```

Comment:
\<nitpick\/\> I think that the `_data` part is implicit i.e. a user can have read access to host_data, remediation_data, advisory_data, subscription_data, ... or we can just say read access to host, remediations, advisory, subscription, ... . I think the latter is cleaner and more in line with other apps.